### PR TITLE
Improve PHP version checking

### PR DESCRIPTION
**Pull request by szepeviktor**
Just 1 line.

# Checklist

- [ ] Project documentation has been updated to reflect the changes in this pull request, if applicable.
- [ ] I have tested the changes in the local development environment (see `contributing.md`).
- [ ] I have added phpunit tests.

### CODE REVIEW

**Comment by derekherman** [comment]
@szepeviktor Not sure I understand the intent of this PR. How is this change an improvement?

**Comment by szepeviktor** [comment]
I inspects the function's return value not its negated value: one less operation.

**Comment by szepeviktor** [comment]
Same as https://github.com/pryley/site-reviews/pull/23

**Review by kidunot89 — APPROVED** [review]

File: stream.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
-if ( ! version_compare( PHP_VERSION, '5.3', '>=' ) ) {
+if ( version_compare( PHP_VERSION, '5.3', '<' ) ) {
```
Comment (kasparsd) [inline stream.php:34]: I personally find the non-negative checks more readable so this looks like a useful change.

**Review by kasparsd — APPROVED** [review]

**Pull request merged**